templates/compute-domain-daemon.tmpl.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: apps/v1
+kind: Daemonset
+metadata:
+  namespace: {{ .Namespace }}
+  generateName: {{ .GenerateName }}
+  finalizers:
+    - {{ .Finalizer }}
+  labels:
+    {{ .ComputeDomainLabelKey }}: {{ .ComputeDomainLabelValue }}
+spec:
+  selector:
+    matchLabels:
+      {{ .ComputeDomainLabelKey }}: {{ .ComputeDomainLabelValue }}
+  template:
+    metadata:
+      labels:
+        {{ .ComputeDomainLabelKey }}: {{ .ComputeDomainLabelValue }}
+    spec:
+      hostNetwork: true
+      nodeSelector:
+        {{ .ComputeDomainLabelKey }}: {{ .ComputeDomainLabelValue }}
+      containers:
+      - name: compute-domain-daemon
+        image: ubuntu:22.04
```

Comment:
What does this resolve to, really (which container registry?)? And where would I find details about that image? (can't quite be vanilla Ubuntu)

Reply:
It is just vanilla ubuntu! The IMEX daemon specifics are injected at runtime by the ResourceClaim that gets referenced.